src/components/Connections/Connections.tsx

```diff
@@ -564,6 +564,7 @@ export const Connections = () => {
         title="Connection"
         headers={headers}
         rows={rows}
+        height={90}
       />
```

Comment:
Actually, the list is still jumping on the `Connections` page. The case you were testing out was a failure one whether the sync didn't start because you created the connection without selecting streams. 

I have created a new connection, you should be able to see for yourself that the list is still jumping.